### PR TITLE
INGK-337 Replace winsocks2 dependency to sockets

### PR DESCRIPTION
**Pull request by MartinALL91**

### CODE REVIEW

File: ingenialink/eth/net.c [hunk 1]
```diff
@@ -328,11 +337,46 @@ static void il_eth_net_close_socket(il_net_t *net) {
...
+    #endif
+
+    return r;	
+}
+
+static int il_eth_net_init_socket(void)
+{
+    #ifdef _WIN32
+        WSADATA wsa_data;
+        return WSAStartup(0x202, &wsa_data);
```
Comment (GullinBustin) [inline ingenialink/eth/net.c:355]: What is 0x202?

File: ingenialink/eth/net.c [hunk 2]
```diff
@@ -391,25 +447,25 @@ static int il_eth_net_is_slave_connected(il_net_t *net, const char *ip) {
...
-                        log_warn("Fail connecting to server");
-                        result = 0;
-                    }
                 }
             }
             else {
-                int last_error = WSAGetLastError();
-                log_warn("Fail connecting to server");
+                int last_error = il_eth_get_last_socket_error();
+                log_warn("Fail connecting to server\n");
```
Comment (GullinBustin) [inline ingenialink/eth/net.c:468]: Did you add the \n in the warning log?

File: ingenialink/eth/net.c (hunk 1 above)
Comment (MartinALL91) [reply ingenialink/eth/net.c:355]: It's the highest version of Windows Sockets support that the caller can use. [Source](http://www.cs.rpi.edu/academics/courses/fall09/os/win32/WSAstartup.html)

File: ingenialink/eth/net.c (hunk 2 above)
Comment (MartinALL91) [reply ingenialink/eth/net.c:468]: No, there were some merge conficts and I made some changes and did not notice the "\n". I will delete it

**Pull request merged**